documents/src/pages/elements/datetime-picker.md

```diff
@@ -662,5 +744,11 @@ ef-datetime-picker .custom-cell.selected {
   aria-labelledby="departure">
 </ef-datetime-picker>
 ```
+```html
+<label for="departure">Enter departure date</label>
```

Comment:
Let's apply this to the above 2 examples as well.
```suggestion
<label for="departure">Departure date</label>
```

Reply:
Fixed!